packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.ts

```diff
@@ -98,6 +99,39 @@ export const getTokenSorterByRef: DesignTokenListTransform = (variableSet) => {
   };
 };
 
+/**
+ * Reorganize the Tokens based on a ordered list of regexps.
+ * Each regexp is applied only to last part of the Token name (before key rendering).
+ * @param regExps Ordered list of regular expressions defining the order of the Tokens.
+ * @param applyRendererName Determine if the regexps are apply to the rendered Token key. If `false`, it will be applied to the Token key's name (last part of the Token name).
```

Comment:
```suggestion
 * Reorganize the Tokens based on an ordered list of regexps.
 * Each regexp is applied only to the last part of the Token name (before key rendering).
 * @param regExps Ordered list of regular expressions defining the order of the Tokens.
 * @param applyRendererName Determine if the regexps are applied to the rendered Token key. If `false`, it will be applied to the Token key's name (last part of the Token name).
```